src/components/Constants.js

```diff
@@ -37,28 +38,28 @@ export default Constants = {
 
   // Individual Badge Badges
   WORLD_COMPLETION_FIRE: 1,
-  WORLD_COMPLETION_FIRE_IMG: require("../assets/badges/world_completion.png"),
-  WORLD_COMPLETION_FIRE_IMG_RAW: "../assets/badges/world_completion.png",
+  WORLD_COMPLETION_FIRE_IMG: require("../assets/badges/lesson_completion.png"),
```

Comment:
You might have to merge the latest version of the code into your branch. This seems a bit outdated